### PR TITLE
Add Sison and Glaz paper.

### PR DESCRIPTION
**Pull request by snim2**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -12,6 +12,19 @@
 
 For 'month' fields, please use the abbreviated 3 letter notation.
 
+@article{sison95simultaneous,
+  title={Simultaneous confidence intervals and sample size determination for multinomial proportions},
```
Comment (ltratt) [inline softdev.bib]: We use an indent of 4 spaces.

File: softdev.bib [hunk 2]
```diff
@@ -12,6 +12,19 @@
...
+  author={Sison, Cristina P and Glaz, Joseph},
```
Comment (ltratt) [inline softdev.bib]: That should probably be either "Cristina P." (with a full stop) or just "Cristina"?

File: softdev.bib [hunk 3]
```diff
@@ -12,6 +12,19 @@
...
+  journal={Journal of the American Statistical Association},
```
Comment (ltratt) [inline softdev.bib]: Shorten to "J.~ASA".

**Comment by snim2** [comment]
OK to squash?

**Comment by ltratt** [comment]
The indent needs to be 4 spaces not 2.

**Comment by ltratt** [comment]
OK, if you squash, I can merge.

**Comment by snim2** [comment]
OK, squashed...

**Pull request merged**